src/features/dashboard/pages/Dashboard.tsx

```diff
@@ -1,3 +1,211 @@
+import { AddTransactionButton } from '@/components/AddTransactionButton';
+import { StatCards } from '../components/StatCards';
+import { PortfolioHistoryChart } from '../components/PortfolioHistoryChart';
+import { DiversificationCharts } from '../components/DiversificationCharts';
+import { columns, StockTableData } from '../components/columns';
+import { DataTable } from '@/components/DataTable';
+
 export const Dashboard = () => {
-  return <>Dashboard</>;
+  //to be replaced with real data later
+  const stockTableData: StockTableData[] = [
```

Comment:
The mock data takes up most of the place in this component, should be in a separate file.